unime/src/lib/components/TermsAndConditions.svelte

```diff
@@ -0,0 +1,76 @@
+<script lang="ts">
+  import LL from '$i18n/i18n-svelte';
+
+  const termsContent = [
+    {
+      title: $LL.ONBOARDING.TERMS.T_AND_C.FULL.AGREEMENT.TITLE(),
+      description: $LL.ONBOARDING.TERMS.T_AND_C.FULL.AGREEMENT.DESCRIPTION(),
+    },
+    {
+      title: $LL.ONBOARDING.TERMS.T_AND_C.FULL.DEFINITIONS.TITLE(),
+      description: $LL.ONBOARDING.TERMS.T_AND_C.FULL.DEFINITIONS.DESCRIPTION(),
+    },
+    {
+      title: $LL.ONBOARDING.TERMS.T_AND_C.FULL.USER_RESPONSIBILITIES.TITLE(),
+      description: $LL.ONBOARDING.TERMS.T_AND_C.FULL.USER_RESPONSIBILITIES.DESCRIPTION(),
+    },
+
+    {
+      title: $LL.ONBOARDING.TERMS.T_AND_C.FULL.DATA_OWNERSHIP.TITLE(),
+      description: $LL.ONBOARDING.TERMS.T_AND_C.FULL.DATA_OWNERSHIP.DESCRIPTION(),
+    },
+    {
+      title: $LL.ONBOARDING.TERMS.T_AND_C.FULL.DATA_VISIBILITY.TITLE(),
+      description: $LL.ONBOARDING.TERMS.T_AND_C.FULL.DATA_VISIBILITY.DESCRIPTION(),
+    },
+    {
+      title: $LL.ONBOARDING.TERMS.T_AND_C.FULL.INTELLECTUAL_PROPERTY_RIGHTS.TITLE(),
+      description: $LL.ONBOARDING.TERMS.T_AND_C.FULL.INTELLECTUAL_PROPERTY_RIGHTS.DESCRIPTION(),
+    },
+    {
+      title: $LL.ONBOARDING.TERMS.T_AND_C.FULL.PROHIBITED_ACTIVITIES.TITLE(),
+      description: $LL.ONBOARDING.TERMS.T_AND_C.FULL.PROHIBITED_ACTIVITIES.DESCRIPTION(),
+    },
+    {
+      title: $LL.ONBOARDING.TERMS.T_AND_C.FULL.THIRD_PARTY_SERVICES.TITLE(),
+      description: $LL.ONBOARDING.TERMS.T_AND_C.FULL.THIRD_PARTY_SERVICES.DESCRIPTION(),
+    },
+    {
+      title: $LL.ONBOARDING.TERMS.T_AND_C.FULL.LIABILITY.TITLE(),
+      description: $LL.ONBOARDING.TERMS.T_AND_C.FULL.LIABILITY.DESCRIPTION(),
+    },
+    {
+      title: $LL.ONBOARDING.TERMS.T_AND_C.FULL.INDEMNIFICATION.TITLE(),
+      description: $LL.ONBOARDING.TERMS.T_AND_C.FULL.INDEMNIFICATION.DESCRIPTION(),
+    },
+    {
+      title: $LL.ONBOARDING.TERMS.T_AND_C.FULL.MODIFICATIONS.TITLE(),
+      description: $LL.ONBOARDING.TERMS.T_AND_C.FULL.MODIFICATIONS.DESCRIPTION(),
+    },
+    {
+      title: $LL.ONBOARDING.TERMS.T_AND_C.FULL.LAW_AND_JURISDIFICATION.TITLE(),
+      description: $LL.ONBOARDING.TERMS.T_AND_C.FULL.LAW_AND_JURISDIFICATION.DESCRIPTION(),
+    },
+    {
+      title: $LL.ONBOARDING.TERMS.T_AND_C.FULL.SEVERABILITY.TITLE(),
+      description: $LL.ONBOARDING.TERMS.T_AND_C.FULL.SEVERABILITY.DESCRIPTION(),
+    },
+    {
+      title: $LL.ONBOARDING.TERMS.T_AND_C.FULL.LANGUAGE.TITLE(),
+      description: $LL.ONBOARDING.TERMS.T_AND_C.FULL.LANGUAGE.DESCRIPTION(),
+    },
+    {
+      title: $LL.ONBOARDING.TERMS.T_AND_C.FULL.ENTIRE_AGREEMENT.TITLE(),
+      description: $LL.ONBOARDING.TERMS.T_AND_C.FULL.ENTIRE_AGREEMENT.DESCRIPTION(),
+    },
+    {
+      title: $LL.ONBOARDING.TERMS.T_AND_C.FULL.CONTACT.TITLE(),
+      description: $LL.ONBOARDING.TERMS.T_AND_C.FULL.CONTACT.DESCRIPTION(),
+    },
+  ];
+</script>
+
+{#each termsContent as section}
```

Comment:
`termsContent` could be renamed to `sections` since it's a common pattern to use:
```
{#each <plural> as <singular>}
```
So like
```
{#each items as item}
or
{#each sections as section}
```